src/components/Dao/DaoPluginsPage.scss

```diff
@@ -1,13 +1,64 @@
 .wrapper {
   width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin-top: -30px; // compensate for containers padding, so position:sticky will work right
 
-  h1 {
-    color: rgba(104, 155, 214, 1);
-    border-bottom: 1px solid rgba(195, 212, 231, 1);
-    width: 100%;
-    padding-top: 30px;
-    background-color: rgba(229, 235, 242, 1);
-    margin: 0;
+  .paddingTop {
+    position: sticky;
+    top: 50px;
+    height: 62px;
+    z-index: 2;
+    background-color: #e5ebf2;
```

Comment:
Please use global variables for colors from `global-variables.scss`.
I know that in many places it's hard-coded but it's a pattern we should eliminate.

Reply:
Also in other places.

Reply:
I totally agree

Reply:
I converted to color and border variables for everything in this file, plus all other files that use the same values.  I didn't address the naming convention of these variables, which can be addressed at a future time.

Reply:
Looks good